pydantic_ai_slim/pydantic_ai/models/openai.py

```diff
@@ -349,7 +349,10 @@ async def _completions_create(
 
     def _process_response(self, response: chat.ChatCompletion) -> ModelResponse:
         """Process a non-streamed response, and prepare a message to return."""
-        timestamp = number_to_datetime(response.created)
+        if response.created:
+            timestamp = number_to_datetime(response.created)
+        else:
+            timestamp = _now_utc()
```

Comment:
Can you please add a test for this scenario where there is no `response.created`? 

There's an equivalent test for `HuggingFaceModel` already:

https://github.com/pydantic/pydantic-ai/blob/6e8c76af8a6c0cc1373b9b1168314090af6ca28c/tests/models/test_huggingface.py#L756-L772

And this test shows how you can similarly fake the `response.created` for `OpenAIModel`:

https://github.com/pydantic/pydantic-ai/blob/6e8c76af8a6c0cc1373b9b1168314090af6ca28c/tests/models/test_openai.py#L1982-L1996